german/the-black-hack.md

```diff
@@ -571,7 +572,7 @@ Anmerkung: Spalten stellen die *Stufe der täglichen Sprüche* dar, Zeilen die *
 **Dieb :** Ich bewege mich dorthin und benutze meine Handlung, um nach Fallen zu suchen.
 **Krieger :** Vorausgesetzt dass alles sicher ist, würde ich versuchen, die Gitterstäbe aufzubiegen.
 **Zauberer :** Und ich möchte Licht auf meinen Stab zaubern.
-**SL :** Ok, Dieb, mach eine Geschickprobe, indem du mit einem W20 deinen GES-Wert unterwürfelst - um das Gitter nach Fallen abzusuchen.
+**SL :** Ok, Dieb, mach eine Geschickprobe, indem du mit einem W20 deinen GES-Wert unterwürfelst, um das Gitter nach Fallen abzusuchen.
```

Comment:
Finde ich unnötig, bzw. sinnverändernd. Ich halte den Teil nach dem Gedankenstrich für einen ergänzenden Nachsatz, zur vorherigen Anweisung.

Vgl.:
> Iss ein Brot, damit du nicht vom Fleisch fällst.

mit:
> Iss ein Brot - damit du nicht vom Fleisch fällst.

Bei dem Spielbeispiel geht es ja um wörtliche Rede und in der zweiten Version ist da für mich zwischen dem erstem und dem zweiten Teil eine Sprechpause. Vielleicht, weil der Sprecher den Satz geistig nicht komplett formuliert hatte, sondern den zweiten Teil nachsetzt, nachdem der erste schon ausgesprochen war.